.gitignore

```diff
@@ -26,4 +26,10 @@
 ###< symfony/web-server-bundle ###
 
 /.php_cs
-/.php_cs.cache
\ No newline at end of file
+/.php_cs.cache
+###> symfony/webpack-encore-bundle ###
+/node_modules/
+/public/build/
+npm-debug.log
+yarn-error.log
```

Comment:
You don't need any `.log` lines here since you have added `*.log`.

Reply:
Resolved